src/components/shared/chart/Canvas.js

```diff
@@ -269,11 +277,20 @@ export class ChartCanvas<HoveredItem> extends React.Component<
 
     const maybeHoveredItem = this.props.hitTest(this._offsetX, this._offsetY);
     if (maybeHoveredItem !== null) {
-      this.setState({
-        hoveredItem: maybeHoveredItem,
-        pageX: event.pageX,
-        pageY: event.pageY,
-      });
+      if (this.state.selectedItem === null) {
+        // Update both the hovered item and the position of the tooltip.
```

Comment:
nit: this comment is useless as it is, please include a "why" we're doing it, or just remove it.

Reply:
It explains that pageX and pageY is the "position of the tooltip", but maybe the wording is not on point. I will explain it better.